coap-chat/main.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2018 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     applications
+ * @{
+ *
+ * @file
+ * @brief       coap-chat - Main loop and shell handlers
+ *
+ * @author      Sebastian Meiling <s@mlng.net>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include "msg.h"
+
+#include "net/gcoap.h"
+#include "kernel_types.h"
+#include "shell.h"
+
+#define BUFLEN          (64U)
+#define MAIN_QUEUE_SIZE (4)
+
+static msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
+
+extern int coap_post(const char *addr, const char *buf, size_t buflen);
+extern void coap_init(void);
+
+int chat(int argc, char **argv)
+{
+    if (argc < 4) {
+        puts("usage: chat <addr> <nick> <msg>");
```

Comment:
You could note that you can specify the interface with \<addr\>.

Reply:
I'll put something in the README